bindings_ffi/src/mls.rs

```diff
@@ -1409,6 +1410,121 @@ mod tests {
         );
     }
 
+
+    #[tokio::test(flavor = "multi_thread", worker_threads = 5)]
+    async fn test_can_stream_group_messages_for_updates() {
+        let alix = new_test_client().await;
+        let bo = new_test_client().await;
+
+        // Stream all group messages
+        let message_callbacks = RustStreamCallback::new();
+        let stream_messages = bo
+            .conversations()
+            .stream_all_messages(Box::new(message_callbacks.clone()))
+            .await
+            .unwrap();
+        tokio::time::sleep(tokio::time::Duration::from_millis(100)).await;
+
+        // Create group and send first message
+        let alix_group = alix
+            .conversations()
+            .create_group(
+                vec![bo.account_address.clone()],
+                FfiCreateGroupOptions::default(),
+            )
+            .await
+            .unwrap();
+
+        tokio::time::sleep(tokio::time::Duration::from_millis(1000)).await;
+
+        alix_group
+            .update_group_name("Old Name".to_string())
+            .await
+            .unwrap();
+
+        let bo_groups = bo
+            .conversations()
+            .list(FfiListConversationsOptions::default())
+            .await
+            .unwrap();
+        let bo_group = &bo_groups[0];
+        bo_group.sync().await.unwrap();
```

Comment:
If you comment out this line the following lines will run and the test will pass